### PR TITLE
Add optical flow

### PR DESCRIPTION
**Pull request by wkentaro**
```bash
python examples/flow.py
```
flow

http://vision.middlebury.edu/flow/data/

### CODE REVIEW

File: imgviz/data/kitti/__init__.py [hunk 1]
```diff
@@ -19,8 +19,36 @@ def read_pose_file(filename):
     return transforms
 
 
+def read_flow(filename):
+    """Read .flo file in Middlebury format"""
+    # Code adapted from:
+    # http://stackoverflow.com/questions/28013200/reading-middlebury-
```
Comment (wkentaro) [inline imgviz/data/kitti/__init__.py]: @KinglittleQ Is this file is related to Kitti?

Comment (wkentaro) [reply imgviz/data/kitti/__init__.py]: Maybe do you have kitti optical flow data?

File: imgviz/flow.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+    return flow_image
+
+
+def flow2rgb(flow_uv, clip_flow=None):
+    """visualize optical flow
+
+    Parameters
+    ----------
+    flow_uv: numpy.ndarray
```
Comment (wkentaro) [inline imgviz/flow.py]: Maybe we should stick to `yx` order ie. `flow_yx`.

**Comment by KinglittleQ** [comment]
I made a mistake, the example data belong to Middlebury dataset.

The u component is corresponding to the horizontal axis.

http://vision.middlebury.edu/flow/code/flow-code/README.txt

File: imgviz/flow.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+    return flow_image
+
+
+def flow2rgb(flow_yx):
+    """visualize optical flow
+
+    Parameters
+    ----------
+    flow_yx: numpy.ndarray
```
[the remark this answers is not in the document]
Comment (wkentaro) [reply imgviz/flow.py]: Ok I updated.
